src/components/cars/Paginate.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactPaginate from 'react-paginate';
+
+const Paginate = ({ pageCount, changePage }) => (
```

Comment:
This is good wotk but we need cars form API, kindly fetch cars from API with image, car_model and description and implement it